### PR TITLE
Adding POI icon for restaurants and fast food

### PR DESCRIPTION
**Pull request by whubsch**
Finally got around to adding an icon for `restaurant`, `fast_food`, and `food_court`. Folks on the Slack channel seemed to agree that combining these, rather than using different icons for `restaurant` and `fast_food`, was preferable. I went back and forth between parallel and diagonal utensils, but I determined that the parallel ones look wonky when there are several restaurants stacked, so to speak. Since they are on the diagonal, there's not much pixel alignment to be done.

Relatedly, I think it would make sense to pursue adding the cuisine tag to OMT (discussion started in openmaptiles/openmaptiles#1527) so that we can differentiate between the various POIs and get a little more visual variety.

This PR makes progress on #692. Open to any feedback or suggestions!

Screenshot 2024-09-11 at 6 27 51 AM
Screenshot 2024-09-11 at 6 24 28 AM

### CODE REVIEW

File: icons/poi_restaurant.svg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<svg height="19.922" width="20" viewBox="0 0 12.245 12.197" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
```
Comment (claysmalley) [inline icons/poi_restaurant.svg:1]:
```suggestion
<svg height="20" width="20" xml:space="preserve" xmlns="http://www.w3.org/2000/svg">
```

File: icons/poi_restaurant.svg [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
...e;stroke-opacity:1;paint-order:markers stroke fill" d="m 8.4543677,1.1039674 c -0.086895,0 -0.1738964,0.032901 -0.2403427,0.099246 L 6.1549695,3.2622689 C 5.6600225,3.757531 5.6171724,4.5479729 5.7774128,5.1805388 L 2.9779019,7.9172649 0.15477181,10.741591 c -0.13860948,0.138407 -0.13860948,0.363779 0,0.502208 l 0.73059402,0.73179 c 0.13860957,0.13843 0.36398187,0.13843 0.50220867,0 L 4.7810701,8.5808978 6.9307839,6.3159116 C 7.5633049,6.4755669 8.3713162,6.4690558 8.8657006,5.9741957 L 10.925952,3.9151402 c 0.132915,-0.1326671 0.132893,-0.3480184 0,-0.4806854 -0.132532,-0.1324648 -0.34698,-0.1324648 -0.47949,0 l -1.83067,1.8294744 c -0.1731605,0.1729805 -0.4534047,0.1729805 -0.6265651,0 -0.1723954,-0.1728454 -0.1723954,-0.452569 0,-0.6253694 L 9.8091354,2.8186513 c 0.137574,-0.1377093 0.137574,-0.3617929 0,-0.4998172 -0.1382269,-0.1375291 -0.3610474,-0.1375291 -0.4986215,0 L 7.4906054,4.1399384 c -0.1731604,0.1731604 -0.4534045,0.1731604 -0.6265651,0 -0.1728005,-0.1727781 -0.1728005,-0.4539223 0,-0.6265651 L 8.6935147,1.6838989 c 0.1328923,-0.1326672 0.1328923,-0.3478383 0,-0.4806855 -0.066266,-0.066344 -0.1522514,-0.099246 -0.239147,-0.099246 z" transform="translate(.61 -.492)"/>
```
Comment (claysmalley) [inline icons/poi_restaurant.svg:3]:
```suggestion
 <path style="stroke:#fff;stroke-width:1.22448;stroke-linejoin:miter;stroke-dasharray:none;stroke-opacity:1;paint-order:markers stroke fill" d="m 0.66217866,1.2798334 c -0.0208431,0 -0.0417421,0.00256 -0.0622027,0.00598 -0.11918445,0.020168 -0.22139548,0.09685 -0.27393118,0.2057477 -0.60301256,1.2463835 -0.35205369,2.7377014 0.62681193,3.7166119 L 2.7041023,6.9582232 c 0.172778,0.1729806 0.4072544,0.2691464 0.6495397,0.2691464 0.025908,0 0.051846,-0.00145 0.077754,-0.00359 l 1.0167748,-0.086125 4.4331393,4.7824324 c 0.068831,0.07435 0.1651714,0.117616 0.2667539,0.119619 h 0.00718 c 0.099309,0 0.1937962,-0.03881 0.2643614,-0.108855 L 10.179196,11.17126 c 0.07015,-0.06999 0.108855,-0.165345 0.108855,-0.26436 0,-0.099 -0.03898,-0.19436 -0.108855,-0.264363 L 0.92534398,1.3898844 C 0.85480118,1.319319 0.76013726,1.2798334 0.66217866,1.2798334 Z"/>
 <path style="stroke:#fff;stroke-width:1.22448;stroke-linejoin:miter;stroke-dasharray:none;stroke-opacity:1;paint-order:markers stroke fill" d="m 8.4543677,1.1039674 c -0.086895,0 -0.1738964,0.032901 -0.2403427,0.099246 L 6.1549695,3.2622689 C 5.6600225,3.757531 5.6171724,4.5479729 5.7774128,5.1805388 L 2.9779019,7.9172649 0.15477181,10.741591 c -0.13860948,0.138407 -0.13860948,0.363779 0,0.502208 l 0.73059402,0.73179 c 0.13860957,0.13843 0.36398187,0.13843 0.50220867,0 L 4.7810701,8.5808978 6.9307839,6.3159116 C 7.5633049,6.4755669 8.3713162,6.4690558 8.8657006,5.9741957 L 10.925952,3.9151402 c 0.132915,-0.1326671 0.132893,-0.3480184 0,-0.4806854 -0.132532,-0.1324648 -0.34698,-0.1324648 -0.47949,0 l -1.83067,1.8294744 c -0.1731605,0.1729805 -0.4534047,0.1729805 -0.6265651,0 -0.1723954,-0.1728454 -0.1723954,-0.452569 0,-0.6253694 L 9.8091354,2.8186513 c 0.137574,-0.1377093 0.137574,-0.3617929 0,-0.4998172 -0.1382269,-0.1375291 -0.3610474,-0.1375291 -0.4986215,0 L 7.4906054,4.1399384 c -0.1731604,0.1731604 -0.4534045,0.1731604 -0.6265651,0 -0.1728005,-0.1727781 -0.1728005,-0.4539223 0,-0.6265651 L 8.6935147,1.6838989 c 0.1328923,-0.1326672 0.1328923,-0.3478383 0,-0.4806855 -0.066266,-0.066344 -0.1522514,-0.099246 -0.239147,-0.099246 z"/>
```

**Review by claysmalley — COMMENTED** [review]
I like the design! This looks good for restaurants, at least until we can start distinguishing them by `cuisine=*`.

The SVG file needs a couple tweaks. Some transform statements snuck in there, and the height isn't a whole number of pixels. Steps to fix:

1. Remove the transforms manually (see suggested changes), then open Inkscape
2. Remove the polygons' strokes temporarily
3. Scale the two polygons together to 18x18 pixels
4. Re-add the strokes with 2px width
5. Move the icons back into place

**Comment by whubsch** [comment]
@claysmalley Those problems should be fixed now but let me know if not.

**Review by claysmalley — APPROVED** [review]
Looks good. Thanks for your contribution!

[image](https://preview.ourmap.us/pr/1153/#map=16.31/41.883717/-87.641664)

**Comment by ZeLonewolf** [comment]
Let's eat!

**Pull request merged**